mapper-runtime/src/main/java/com/datastax/oss/driver/api/mapper/annotations/Entity.java

```diff
@@ -74,4 +74,24 @@
  */
 @Target(ElementType.TYPE)
 @Retention(RetentionPolicy.CLASS)
-public @interface Entity {}
+public @interface Entity {
+  /**
+   * Specifies a default keyspace to use when doing operations on a given {@code @Entity}
+   *
+   * <p>The defaultKeyspace is ignored if you will create your {@link Dao} using {@link DaoFactory}
+   * with explicitly setting {@link DaoKeyspace}:
+   *
+   * <pre>
+   *  &#64;Mapper
+   *  public interface InventoryMapper {
+   *    &#64;DaoFactory
+   *    ProductDao productDao(@DaoKeyspace String keyspace);
+   *  }
+   * </pre>
```

Comment:
This might be obvious, but just to drive the message home I would make the example more descriptive:

Add an outline of the entity definition before the mapper:
```java
@Product(defaultKeyspace = "inventory")
public class Product { ... }
```

Add a parameterless factory method to the mapper:
```java
@DaoFactory
ProductDao productDao();
```

Add sample client code:
```java
ProductDao productDao = mapper.productDao();
productDao.insert(product); // inserts into inventory.product

ProductDao productDaoTest = mapper.productDao("test");
productDaoTest.insert(product); // inserts into test.product
```